oci/tests/integration/Makefile

```diff
@@ -6,14 +6,14 @@ TEST_IMG ?= fluxcd/testapp:test
 
 .PHONY: app
 app:
-	CGO_ENABLED=0 go build -v -o app ./testapp
+	CGO_ENABLED=0 GOOS=linux go build -v -o app ./testapp
 
 docker-build: app
-	docker build -t $(TEST_IMG) --load .
+	docker build -t $(TEST_IMG) .
 
 test:
 	docker image inspect $(TEST_IMG) >/dev/null
-	TEST_IMG=$(TEST_IMG) go test -timeout $(TEST_TIMEOUT) -v ./ $(GO_TEST_ARGS) $(PROVIDER_ARG) --tags=integration
+	TEST_IMG=$(TEST_IMG) go test -timeout $(TEST_TIMEOUT) -v ./ $(PROVIDER_ARG) $(GO_TEST_ARGS) --tags=integration
```

Comment:
Was this causing any problem?

Reply:
I don't remember any. I will undo the change